packages/amplication-prisma-db/prisma/schema.prisma

```diff
@@ -18,17 +18,18 @@ datasource db {
 }
 
 model Account {
-  id            String   @id @default(cuid())
-  createdAt     DateTime @default(now())
-  updatedAt     DateTime @updatedAt
-  email         String   @unique(map: "Account.email_unique")
-  firstName     String
-  lastName      String
-  password      String
-  currentUserId String?  @unique(map: "Account.currentUserId_unique")
-  githubId      String?
-  currentUser   User?    @relation(fields: [currentUserId], references: [id])
-  users         User[]   @relation("AccountOnUser")
+  id             String   @id @default(cuid())
+  createdAt      DateTime @default(now())
+  updatedAt      DateTime @updatedAt
+  email          String   @unique(map: "Account.email_unique")
+  firstName      String
+  lastName       String
+  password       String
+  currentUserId  String?  @unique(map: "Account.currentUserId_unique")
+  githubId       String?
+  currentUser    User?    @relation(fields: [currentUserId], references: [id])
+  users          User[]   @relation("AccountOnUser")
+  notificationId String?
```

Comment:
What is the purpose of this new column? If it is the notification system user id, should it not be on the notification-service side?
Server entity `Account` should be not contain other systems data

Reply:
this is the id for any account that will be register on Novu. I need it here for two reasons: one, security issue (I don't want to expose our users ids), second, it will be used by our client to register user for receiving notifications

Reply:
But should not be the notification-service to hold that mapping `userId<->notificationSystemId`?